.github/workflows/nightlies.yml

```diff
@@ -38,8 +38,9 @@ jobs:
       - uses: goreleaser/goreleaser-action@v6
         with:
           distribution: goreleaser-pro
-          version: v1.26.2-pro
-          args: release --clean --nightly --config ./.github/goreleaser-nightly.yaml
+          version: ~> v2
```

Comment:
Dumb question, but do we need to put "pro"?

Reply:
I think this was dropped starting from `goreleaser-action@v6` for v2, where it is enough `distribution: goreleaser-pro`